### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by nydame** [openoakland/openbudgetoakland#139]
Jess, a member of Open Oakland pointed out that people who cannot distinguish red and green may have trouble with the colors we've chosen. There may be lots of other issues that we aren't even aware of. This audit should be easy to do with the accessibility tab in Firefox (and Chrome?).

Deliverable: instead of a pull request, submit a report in our wiki.

**Comment by hedera46** [comment]
After one look at your front page, I have a comment for this issue.  Where is your wiki?  A link to the wiki in this issue would be very helpful.  

For the record, here's my first comment:  I'm 74 years old and wear glasses, which help but have limits.  The 2 graphs under "Focus on Measure Z" are unreadable in their normal state, for me and a lot of other older and vision-impaired folks:  the fonts are too small, and the contrast between the font colors and the background is not great enough.  Black text fonts, and larger, if you please.  The older we get, the more trouble we have with low-contrast small text.  I admit that I found I can read them better by clicking on the photo, which makes it Much Larger and perfectly readable; but I'm a retired programmer with over 20 years experience dealing with computer displays and various sorts of web site displays.  Consider that an aging person with poor eyesight and little to no experience with web sites may not even realize that clicking on the display would give them a nice enlargement.  How about a "click here for better detail?"  That would probably work as a substitute for redoing all the fonts.

**Comment by nydame** [comment]
...and here is a link to our wiki: https://github.com/openoakland/openbudgetoakland/wiki
